docs/en/observability/profiling-troubleshooting.asciidoc

```diff
@@ -0,0 +1,225 @@
+[[profiling-troubleshooting]]
+= Troubleshoot your Universal Profiling 
+
+++++
+<titleabbrev>Troubleshoot</titleabbrev>
+++++
+
+beta::[]
+
+You can use the host-agent logs to find errors.
+
+The following is an example of a _healthy_ host-agent output:
+
+----

```

Comment:
All of the code blocks should specify a source: This differentiates a block a from a code block and will enable code highlighting (if applicable).

```suggestion
[source,logs]
----

```